module/VuFind/src/VuFind/Controller/SolrAlternativeController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * SolrAlternative Controller
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Staats- und Universitätsbibliothek Hamburg 2018.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Search_SolrAlternative
+ * @author   Hajo Seng <hajo.seng@sub.uni-hamburg.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFind\Controller;
+
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+/**
+ * SolrAlternative Controller
+ *
+ * @category VuFind
+ * @package  Search_SolrAlternative
+ * @author   Hajo Seng <hajo.seng@sub.uni-hamburg.de>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+class SolrAlternativeController extends AbstractSearch
+{
+    /**
+     * Constructor
+     *
+     * @param ServiceLocatorInterface $sm Service locator
+     */
+    public function __construct(ServiceLocatorInterface $sm)
+    {
+        $this->searchClassId = 'SolrAlternative';
+        parent::__construct($sm);
+    }
+
+    /**
+     * Home action
+     *
+     * @return mixed
+     */
+    public function homeAction()
+    {
+        return $this->createViewModel();
+    }
+    /**
+     * Is the result scroller active?
+     *
+     * @return bool
+     */
+    protected function resultScrollerActive()
+    {
+        $config = $this->serviceLocator->get('VuFind\Config')
+            ->get('SolrAlternative');
+        return isset($config->Record->next_prev_navigation)
+            && $config->Record->next_prev_navigation;
+    }
+
+    /**
+     * Search action -- call standard results action
+     *
+     * @return mixed
+     */
+    public function searchAction()
```

Comment:
Re-routing the search action to the results action was originally done for legacy compatibility with VuFind 1.x. You could simplify things by removing searchAction and changing the route name from /Search to /Results above.

Reply:
I'm going to have a closer look at this